res/bundle/strings_se.properties

```diff
@@ -0,0 +1,4 @@
+﻿# Svensk lokalisering. Do not remove first comment line, coz app dont see first line key for no reason and crash....menu.bar.newPlaylist=Ny spellista
```

Comment:
Swedish localization is sv not se.
I tested your PR and it first didn't work but after i wrote getResourseBundle("sv") I finally got the Swedish translation.

Reply:
Ops. Want me to fix it or you implement code youself?

Reply:
You can fix it, It's a small change and I have some other things that needs to be done :)

Reply:
okie

Reply:
You can override Sysytem user.language by run application with vm option "-Duser.language=sv"
new commit on the way

Reply:
You mean for testing? Like if I want to test the russian translation I would do "-Duser.language=ru" right? For me my System has the default language set to sv.